somaticseq/run_somaticseq.py

```diff
@@ -0,0 +1,440 @@
+
+
+    # Classify SNV calls
+    if classifier_snv:
+        classifiedSnvTsv = outdir + os.sep + classifiedOutPrefix + 'sSNV.tsv'
+        classifiedSnvVcf = outdir + os.sep + classifiedOutPrefix + 'sSNV.vcf'
+
+        subprocess.call( (adaPredictor, classifier_snv, ensembleSnv, classifiedSnvTsv) )
+
+        tsv2vcf.tsv2vcf(classifiedSnvTsv, classifiedSnvVcf, snvCallers, pass_score=pass_threshold, lowqual_score=lowqual_threshold, hom_threshold=hom_threshold, het_threshold=het_threshold, single_mode=True, paired_mode=False, tumor_sample_name=sample_name, print_reject=True, phred_scaled=True)
+
+
+    else:
+        # Train SNV classifier:
+        if somaticseq_train and truth_snv:
+            subprocess.call( (adaTrainer, ensembleSnv, 'Consistent_Mates', 'Inconsistent_Mates') )
+
+
+        consensusSnvVcf = outdir + os.sep + consensusOutPrefix + 'sSNV.vcf'
+        tsv2vcf.tsv2vcf(ensembleSnv, consensusSnvVcf, snvCallers, hom_threshold=hom_threshold, het_threshold=het_threshold, single_mode=True, paired_mode=False, tumor_sample_name=sample_name, print_reject=True)
+
+
+
+    ###################### INDEL ######################
+    single_sample_vcf2tsv.vcf2tsv(is_vcf=outIndel, bam_fn=bam, truth=truth_indel, cosmic=cosmic, dbsnp=dbsnp, mutect=intermediateVcfs['MuTect2']['indel'], varscan=intermediateVcfs['VarScan2']['indel'], vardict=intermediateVcfs['VarDict']['indel'], lofreq=intermediateVcfs['LoFreq']['indel'], scalpel=scalpel, strelka=intermediateVcfs['Strelka']['indel'], dedup=True, min_mq=min_mq, min_bq=min_bq, min_caller=min_caller, ref_fa=ref, p_scale=None, outfile=ensembleIndel)
+
+
+    # Classify INDEL calls
+    if classifier_indel:
+        classifiedIndelTsv = outdir + os.sep + classifiedOutPrefix + 'sINDEL.tsv'
+        classifiedIndelVcf = outdir + os.sep + classifiedOutPrefix + 'sINDEL.vcf'
+
+        subprocess.call( (adaPredictor, classifier_indel, ensembleIndel, classifiedIndelTsv) )
+
+        tsv2vcf.tsv2vcf(classifiedIndelTsv, classifiedIndelVcf, indelCallers, pass_score=pass_threshold, lowqual_score=lowqual_threshold, hom_threshold=hom_threshold, het_threshold=het_threshold, single_mode=True, paired_mode=False, tumor_sample_name=sample_name, print_reject=True, phred_scaled=True)
+
+    else:
+        # Train INDEL classifier:
+        if somaticseq_train and truth_indel:
+            subprocess.call( (adaTrainer, ensembleIndel, 'Strelka_QSS', 'Strelka_TQSS', 'Consistent_Mates', 'Inconsistent_Mates') )
+
+
+        consensusIndelVcf = outdir + os.sep + consensusOutPrefix + 'sINDEL.vcf'
+        tsv2vcf.tsv2vcf(ensembleIndel, consensusIndelVcf, indelCallers, hom_threshold=hom_threshold, het_threshold=het_threshold, single_mode=True, paired_mode=False, tumor_sample_name=sample_name, print_reject=True)
+
+
+
+    ## Clean up after yourself ##
+    if not keep_intermediates:
+        for file_i in files_to_delete:
+            subprocess.call( ('rm', '-v', file_i ) )
```

Comment:
`os.remove( )`?

Reply:
An easy way to print out files that are removed.